django_app/tests_playwright/_signin.py

```diff
@@ -0,0 +1,26 @@
+import os
+import subprocess
+from playwright.sync_api import Page, expect
+
+def sign_in(page: Page):
+
+    print("Signing in")
+
+    email_address = os.environ["USER_EMAIL"]
+
+    # Sign in page
+    page.goto("http://localhost:8090/sign-in/")
+    expect(page.get_by_text("Redbox Copilot")).to_be_visible()
+    page.get_by_label("Email Address").type(email_address)
+    page.get_by_text("Continue").click()
+
+    # Get magic link
+    os.chdir('..')
+    command = ["poetry", "run", "python", "manage.py", "show_magiclink_url", email_address]
+    result = subprocess.run(command, capture_output=True, text=True)
```

Comment:
Because the working dir for this is `tests_playwright` it felt safest to navigate one step back from there, rather than outwards in (if that makes sense) - i.e. `tests_playwright` will always be an immediate child dir of `django_app` unless we explicitly move these tests. And so it should work regardless of what the overall working dir is. Unless I've missed something?

Reply:
Ah, with you, thanks @wpfl-dbt. That's now updated!